cli/tools/task.rs

```diff
@@ -314,6 +335,48 @@ fn print_available_tasks(
   Ok(())
 }
 
+struct NpmCommand;
+
+impl ShellCommand for NpmCommand {
+  fn execute(
+    &self,
+    mut context: ShellCommandContext,
+  ) -> LocalBoxFuture<'static, ExecuteResult> {
+    if context.args.first().map(|s| s.as_str()) == Some("run")
+      && !context.args.iter().any(|s| s == "--")
+    {
+      if let Some(task_name) = context.args.get(1) {
```

Comment:
I wonder if we should try to unit test this. WDYT?

Reply:
Done.